js/mobile-menu.js

```diff
@@ -0,0 +1,35 @@
+function reloadBrowser() {
+  // eslint-disable-next-line no-restricted-globals
+  location.reload();
+}
+
+const portfolioElement = document.getElementById('portfolio');
+portfolioElement.addEventListener('click', () => {
+  document.getElementById('nav-mobile').style.display = 'none';
+  // eslint-disable-next-line no-use-before-define
+  reloadBrowser();
+});
+
+const aboutElement = document.getElementById('about');
+aboutElement.addEventListener('click', () => {
+  document.getElementById('nav-mobile').style.display = 'none';
+  // eslint-disable-next-line no-use-before-define
+  reloadBrowser();
+});
+
+const contactElement = document.getElementById('contact');
+contactElement.addEventListener('click', () => {
+  document.getElementById('nav-mobile').style.display = 'none';
+  // eslint-disable-next-line no-use-before-define
+  reloadBrowser();
+});
+
+const mobilemenu = document.getElementById('open-menu');
+mobilemenu.addEventListener('click', () => {
+  document.getElementById('nav-mobile').style.width = '100%';
+});
+
+const closebtn = document.getElementById('closebtn');
+closebtn.addEventListener('click', () => {
+  document.getElementById('nav-mobile').style.width = '0%';
```

Comment:
- please as it is intended to work, when I click on the menu links in the mobile view, it should navigate me to the appropriate section in your webpage, but on your design, it closes the menu without navigating me to the current page.
Hint: Try adding an `a tag to the menu and also on the section head you want it to link, and add on your CSS body. a smooth scroll property so that it scrolls smoothly when navigating the page.

##### Section on your design 
![Screenshot from 2022-06-23 07-41-24](https://user-images.githubusercontent.com/58771507/175233019-c615090c-cd11-42d4-8458-64beea52e45d.png)

Reply:
Done as advised